clients-simple/src/main/resources/application-eureka.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+  eureka:
+    instance:
+      hostname: localhost
+      port: 8083
+      preferIpAddress: false
+      lease-renewal-interval-in-seconds: 5
+      metadataMap:
+        instanceId: ${vcap.application.instance_id:${spring.application.name}:${spring.application.instance_id:${server.port}}}
+    client:
+      serviceUrl:
+        defaultZone: http://${eureka.instance.hostname}:${eureka.instance.port}/eureka/
+    enabled: true
+    registerWithEureka: false
+    fetchRegistry: true
+    registry-fetch-interval-seconds: 5
+  web:
+    client:
+      palisade-service: palisade-service
+      data-service: data-service
```

Comment:
After adding url=${web.client.xxx-service} to the Feign builder, these should be ""